lib/Rex/Interface/Shell/Ilo.pm

```diff
@@ -0,0 +1,47 @@
+#
+# (c) Ferenc Erki <erkiferenc@gmail.com>, adjust GmbH
```

Comment:
I did not author this code. Please make sure that the copyright is properly attributed.

Reply:
done, see https://github.com/RexOps/Rex/pull/1468#issuecomment-784166593.